src/Behat/Behat/Definition/Pattern/Policy/TurnipPatternPolicy.php

```diff
@@ -132,11 +133,22 @@ private function replaceTokensWithRegexCaptureGroups($regex)
 
         return preg_replace_callback(
             self::PLACEHOLDER_REGEXP,
-            function ($match) use ($tokenRegex) { return sprintf($tokenRegex, $match[1]); },
+            array($this, 'replaceTokenWithRegexCaptureGroup'),
             $regex
         );
     }
 
+    private function replaceTokenWithRegexCaptureGroup($tokenMatch)
+    {
+        if (mb_strlen($tokenMatch[1]) >= 32) {
```

Comment:
does PHP really allows for multibyte chars in capture group names, and handles them properly ? I don't think so. I'm quite sure the limit is 32 single-byte chars. So you should use ``strlen``